db/migrate/20230105160357_rename_flipper_admin_access_to_superadmin.rb

```diff
@@ -0,0 +1,7 @@
+class RenameFlipperAdminAccessToSuperadmin < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :super_admin, :boolean, default: false, null: false
+
+    User.where(flipper_admin_access: true).update_all(super_admin: true)
```

Comment:
Is flipper access used anywhere else? if not maybe we can delete the old column in this migration?

Reply:
I've got a follow up PR to do that, the migration runs before the deployment so we'd have a moment where a column may be accessed that no longer exists. This PR deprecates then a follow up removes the no longer used column.